src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenInParametersTests.cs

```diff
@@ -1165,6 +1182,21 @@ public void M1(in int arg2, in int arg3, in int arg4)
 42
 3
 3");
+
+            comp = CreateCompilationWithMscorlib46(text, new[] { ValueTupleRef, SystemRuntimeFacadeRef }, options: TestOptions.ReleaseExe, parseOptions: TestOptions.Regular.WithPEVerifyCompatFeature());
+            CompileAndVerify(comp, verify: false, expectedOutput: @"
+3
+42
+2
```

Comment:
Basically - if you pass a readonly field to an `in` parameter do you want:
1 - never pass it by reference (defeats the purpose of the `in`, definitely no )
2 - always pass it by reference (triggers peverify complains)
3 - change the behavior based on the compat flag (some contorted code may see the difference)

We have `#3` right now.